.github/workflows/unitctl.yml

```diff
@@ -9,7 +9,7 @@ on:
     branches:
       - master
     tags:
-      - unitctl/[0-9]+.[0-9]+.[0-9]+
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
I'm not sure exactly what regex syntax this is using but for example with PCRE that would also match the packaging tags (_if_ it matches against lightweight tags) e.g

```
$ echo "1.32.1-1" | grep -P '[0-9]+.[0-9]+.[0-9]+'
1.32.1-1
```

with PCRE at least you'd want `^[0-9]+\.[0-9]+\.[0-9]+$`

E.g
```
$ echo "1.32.1-1" | grep -P '^[0-9]+\.[0-9]+\.[0-9]+$'
$
```
We also want to _match_ the `.`'s, not any character... (though looking at the GH docs is _looks_ like `.` doesn't have that meaning here...)

(in our case the `^` is technically superfluous, but doesn't hurt and at this point we'd technically only need to check if the first digit is [1-9])

Reply:
Its not real regex, its Github's special subset of regex. I did test using a more robust regex:
```
([0-9]+\.[0-9]+\.[0-9]+)(-[0-9]+)?
```
But found during testing that this wasnt actually regex that this was parsing.